livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/utils.py

```diff
@@ -0,0 +1,7 @@
+import os
```

Comment:
```suggestion
from __future__ import annotations

import os
```

Reply:
ah i just started using `Optional`.. will keep in mind in the future. def seems easier